doc/doxyfile.in

```diff
@@ -44,14 +44,14 @@ PROJECT_NUMBER         = @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = 
+PROJECT_BRIEF          =
```

Comment:
What is generated by doxygen is the _template_. We still need to edit it ourselves, so I don't consider the final file a generated file. What I was referring to as the helper program is stuff like doxywizard, a program that lets you edit/modify the file in a GUI, but I don't see why we would need to use keep using it to make any necessary future changes to the file if we can just change the lines ourself.